sorc/ncep_post.fd/FIXED.f

```diff
@@ -109,7 +110,7 @@ SUBROUTINE FIXED
            DO I = 1,IM
              GRID1(I,J) = SPVAL
               IF(SM(I,J)   /= SPVAL) GRID1(I,J) = 1. - SM(I,J)
-              IF(SICE(I,J) /= SPVAL .AND. SICE(I,J) > 0.1) GRID1(I,J) = 0.
+              IF(SICE(I,J) /= SPVAL .AND. SICE(I,J) > 0.0) GRID1(I,J) = 0.
```

Comment:
@HuiyaChuang-NOAA The UPP regression testing indicates this change having impact on rap/hrrr/3drtma land-sea mask. Would you consider to add condition as:
IF (modelname = "GSF" or modelname == "FV3R") then
   if(sice(i,j)/=spaval .and. sice(i,j) > 0.) grid1(i,j)=0.
else
   if(sice(i,j)/=spavel .and. sice(i,j) > 0.1) grid1(i,j)=0.
endif

Reply:
@WenMeng-NOAA are you seeing the land mask in GFS?  If rap/hrrr/rtma uses binary sea ice, you shouldn't see a change in land mask unless sice is a very small number instead of zero for a grid point without sea ice.  I am thinking using 
"SMALL=1.E-6" instead of zero.  Are you willing to give this another try?  If you really need to get back to your wcoss2 transition work, I will make the changes as you suggested.

Reply:
@WenMeng-NOAA made the update per your suggestion.  Ran a test to ensure new land mask and foundation T
looked fine.  Just committed the changes to my branch.

Reply:
@HuiyaChuang-NOAA I don't see recent changes in your branch Add_tref.

Reply:
@WenMeng-NOAA sorry only made local commit but just pushed it to repo

Reply:
@HuiyaChuang-NOAA Now it looks good to me. I will conduct regression tests. Thanks!